CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ You may also find the [Upgrade Guide](https://rust-random.github.io/book/update.
 - Implement weighted sampling without replacement (#976, #1013)
 
 ### Changes
+- `getrandom` updated to v0.2
```

Comment:
This could mention this PR by issue number.

Reply:
Done